shard-operator/README.md

```diff
@@ -89,3 +89,4 @@ Status:
   Status:  OK
 Events:    <none>
 ```
+
```

Comment:
Is this intentional?

Reply:
it was used to trigger PR check, not needed but I didn't remove it as it is harmless :)